src/workloads/sharding/WouldChangeOwningShardBatchWrite.yml

```diff
@@ -0,0 +1,93 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+Description: |
+  Creates a sharded collection with 2 chunks on 2 different shards using ranged sharding and updates
+  the shard key to trigger WouldChangeOwningShard errors.
+
+  The workload consists of 3 phases:
+    1. Shard an empty collection (using ranged sharding) into two chunks and wait for the balancer 
+    to distribute each chunk to its own shard.
+    2. Populate the sharded collection with data.
+    3. Update the shard key value to trigger WouldChangeOwningShard errors.
```

Comment:
So the goal of this workload is to trigger these errors? If so you might want to configure the throw settings for CrudActor like [here](https://github.com/mongodb/genny/blob/5fdb13cc6d9b6153fd2567e9d536f509289a5725/src/workloads/docs/CrudActor.yml#L39). Otherwise the workload may crash out.

Reply:
The error itself is implicitly retried on so the client doesn't actually see that we throw WouldChangeOwningShard.

Reply:
Ah gotcha, thanks for clarifying!